Library/Homebrew/test/cask/audit_spec.rb

```diff
@@ -91,6 +91,25 @@ def include_msg?(messages, msg)
       end
     end
 
+    describe "pkg allow_untrusted checks" do
+      let(:error_msg) { "allow_untrusted is not permitted in official Homebrew-Cask taps" }
+
+      context "when the cask has no pkg stanza" do
+        let(:cask_token) { "basic-cask" }
+        it { should_not warn_with(error_msg) }
+      end
+
+      context "when the cask does not have allow_untrusted" do
+        let(:cask_token) { "with-uninstall-pkgutil" }
+        it { should_not warn_with(error_msg) }
+      end
+
+      context "when the cask has allow_untrusted" do
```

Comment:
Uppercase “Cask” in these.

Reply:
Lowercase is consistent with the other checks.

Reply:
I generally try to capitalize Homebrew terminology like Tap, Formula, Cask etc. so they stand out a bit.